### PR TITLE
doc: explain how to remove reviewers

### PR DESCRIPTION
**Pull request by sileht**
Closes: #88

### CODE REVIEW

File: doc/source/configuration.rst [hunk 1]
```diff
@@ -172,7 +172,8 @@ keys:
    * - ``required_approving_review_count``
      - ``integer``
      - Specify the number of reviewers required to approve pull requests. Use a
-       number between 1 and 6.
+       number between 1 and 6. If you don't want any reviewer, set
```
Comment (jd) [inline doc/source/configuration.rst]: s/want/need/

File: doc/source/getting-started.rst [hunk 2]
```diff
@@ -79,6 +79,26 @@ multiple branches at once, you can use a regular expression:
 In this example, pull requests sent to a branch whose name matches ``stable/*``
 will only require one review approval and will still require Travis to pass.
 
+You also can automatically merge PR as soon all your required status checks
```
Comment (jd) [inline doc/source/getting-started.rst]: You can automatically merge a pull request as soon as all the required status checks pass:

File: doc/source/getting-started.rst [hunk 3]
```diff
@@ -79,6 +79,26 @@ multiple branches at once, you can use a regular expression:
...
+          contexts:
+            - continuous-integration/travis-ci
+        required_pull_request_reviews:
+          required_approving_review_count: 2
+      branches:
+        protection:
+          auto-deployment:
+            required_pull_request_reviews: null
+
+This times, pull requests sent to branch "auto-deployement" will require
```
Comment (jd) [inline doc/source/getting-started.rst]: With that configuration, pull requests sent to the branch ``auto-deployment`` require no approvals, while still requiring Travis continuous integration to succeed.

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 004a656a937170e6bc6db5915729305d206fbb01

**Pull request merged**